thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -1,6 +1,13 @@
 import Ember from 'ember';
 import { toBaselineUrn, hasPrefix, filterPrefix } from '../../../helpers/utils';
 
+const ROOTCAUSE_METRICS_SORT_PROPERTY_METRIC = 'metric';
```

Comment:
I feel that having this in one const PROP will be more readable: 
```
const ROOTCAUSE_METRICS_SORT_PROPERTY = {
  DATASET= 'dataset',
  CHANGE= 'change',
  METRIC='metric'
}
```

Reply:
hmm. the definition yes. the comparison will look worse "ROOTCAUSE_METRICS_SORT_PROPERTY.DATASET". isn't there a better way to do enums in JS?

Reply:
not with Vanilla JS. But typescript allows for it: 
https://www.typescriptlang.org/docs/handbook/enums.html